exercises/1/build/build.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
```

Comment:
The moral-message of this task:
```
:star: Your PR does not contain unrelated changes
- Double check "Files changed" tab in your pull request
```
was to get empathy for your code reviewer :laughing:

You're asking your co-worker to look at your changed files. *This takes time.* As the person who is requesting a code review, you should learn to put only relevant changes in the PR.

In your job, you will receive the best code reviews when your "Files changed" are *small*.